src/authentication/Logout.tsx

```diff
@@ -8,11 +8,16 @@ import {logout} from '../actions/AuthActions';
  * @param props auth state from redux
  * @constructor
  */
-const Logout = (props : any) => (
-    <div><Redirect to={{
-        pathname: '/'
-    }}/></div>
-);
+const Logout = (props : any) => {
+    //redirects the user to the main page and logs him out
+    props.logout()
+
+    return (
+        <div><Redirect to={{
```

Comment:
divje verwijderen

Reply:
En constant maken op andere plekken zoals in de Login.tsx

Reply:
Implemented in : commit:
edbc9a5